assets/scss/modules/typography/_lists.scss

```diff
@@ -1,17 +1,17 @@
 ul, ol {
   margin: $leading 0 $leading*2 0;
-  padding-left: 1em;
-  list-style-type: square;
+  padding-left: 0px;
```

Comment:
This is what's wiped out the bullets - leave it at `padding-left: 1em;` ?

As an aside, `0` is fine, you don't need to write `0px`